downstream/titles/release-notes/topics/eda-24.adoc

```diff
@@ -0,0 +1,61 @@
+// This is the release notes for Event-Driven Ansible for AAP 2.4 release, the version number is removed from the topic title as part of the release notes restructuring efforts.
+
+[[eda-24-intro]]
+= Event-Driven Ansible
+
+Event-Driven Ansible is the newest capability of Ansible Automation Platform that is designed to enable automated response with user-defined, rules-based workflows. Event-Driven Ansible works by receiving events from third party tools, deciding on the actions to take and acting automatically.  
```

Comment:
```suggestion
Event-Driven Ansible is the newest capability of Ansible Automation Platform that is designed to enable automated response with user-defined, rules-based workflows. Event-Driven Ansible works by receiving events from third party tools, deciding on the actions to take, and acting automatically.  
```
IBM style guide (https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series)